tests/testthat/test-modify-model-field.R

```diff
@@ -354,3 +354,50 @@ test_that("remove_tags() works correctly [BBR-MMF-022]", {
   )
   expect_identical(new_mod[[YAML_TAGS]], ref_tags)
 })
+
+test_that("Checking that the star_add function is adding boolean data to YAML[BBR-MMF-023]", {
+  temp_mod_path <- create_temp_model()
+
+  # make a model object and check YAML
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- add_star(new_mod, TRUE)
+
+  expect_true(new_mod$starred)
+
+
+})
+
+test_that("Checking that the star_remove function is removing boolean data from YAML[BBR-MMF-023]", {
+  temp_mod_path <- create_temp_model()
+
+
+  # make a model object and replace the tags
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- add_star(new_mod, TRUE)
+
+
+  new_mod <- remove_star(new_mod, FALSE) %>% suppressWarnings()
+  expect_true(new_mod$starred)
+
+  new_mod <- remove_star(new_mod, TRUE)
+  expect_failure(expect_true(new_mod$starred))
```

Comment:
`expect_failure(expect_true(...))` -> `expect_false(...)`?

Reply:
I was getting logical(0). It is not FALSE or TRUE.

Reply:
I think it may be better to consolidate add star and remove star into one function. It might be best to rename replace_start with set_star_status to better reflect the goal of the function.

Reply:
It feels like this comment is in the wrong thread, but I think this is in reply to my suggestion in a few other spots that the `modify_model_field` wrappers for star could be

```
add_star(.mod)
remove_star(.mod)
```

> It might be best to rename replace_start with set_star_status to better reflect the goal of the function

Hmm, I don't know, it seems nice to me to stick with the `remove_` and `add_` used by the other `modify_model_field` wrappers.  But I'm perhaps not the best person to say.  It might be worth seeing what @seth127 thinks.